docs/manager/README.md

```diff
@@ -1049,21 +1050,12 @@ In case if AccelerateWP Premium is active, the auxiliary monitoring daemon `clwp
 ### FAQ
 
 #### With which panel can I use AccelerateWP?
-Currently AccelerateWP is compatible with cPanel only.
+For now, AccelerateWP is compatible with cPanel and Plesk.
 
-If you are interested in supporting Plesk or DirectAdmin control panels, please record your demand on our feature portal respectively:
-* [AccelerateWP for Plesk](https://features.cloudlinux.com/c/13-acceleratewp-for-plesk?utm_medium=social&utm_source=portal_share)
-* [AccelerateWP for DirectAdmin](https://features.cloudlinux.com/c/15-acceleratewp-for-directadmin?utm_medium=social&utm_source=portal_share)
-
-#### Does WP Accelerate tool support the DirectAdmin control panel? Do you plan to integrate it with Plesk?
-Currently AccelerateWP is compatible with cPanel only.
-
-If you are interested in supporting Plesk or DirectAdmin control panels, please record your demand on our feature portal respectively:
-* [AccelerateWP for Plesk](https://features.cloudlinux.com/c/13-acceleratewp-for-plesk?utm_medium=social&utm_source=portal_share)
-* [AccelerateWP for DirectAdmin](https://features.cloudlinux.com/c/15-acceleratewp-for-directadmin?utm_medium=social&utm_source=portal_share)
+If you want DirectAdmin to be supported, please submit your request to our feature portal [AccelerateWP for DirectAdmin](https://features.cloudlinux.com/c/15-acceleratewp-for-directadmin?utm_medium=social&utm_source=portal_share).
```

Comment:
Please, leave the previous text as is, just remove Plesk info.
As far as I can remember, the text was written and approved by Dennis.

Reply:
thank you